README.md

```diff
@@ -1,10 +1,14 @@
 # Dock Wallet SDK
 
-Using [polkadot-js](https://polkadot.js.org/) libraries in React Native is a challenge, due to a lack of WebAssembly support.
+The [Wallet SDK](https://github.com/docknetwork/react-native-sdk) enables you to build a Verifiable Credentials wallet inside your app and allows your users to receive, store, and manage their DOCK tokens too. This was built for native applications with added support for Polkadot-JS.
```

Comment:
what are we trying say with the statement, `This was built for native applications...` In the mobile world a native application is usually written in Swift or Kotlin depending on the platform.